Yoast/wordpress-seo
# Removed recalibration beta toggle

## Summary

This PR can be summarized in the following changelog entry:

* [not-userfacing] Removed recalibration beta toggle.

## Relevant technical choices:

* Deprecated `WPSEO_Recalibration_Beta`.
* Removed ability to retrieve recalibration files from yoast.com.
* Added upgrade routine to delete recalibration beta option.
* Removed the webpack configuration for a separate recalibration build.
* Removed the feature flag for using local recalibration file.

## Test instructions

This PR can be tested by following these steps:

* Make sure that the recalibration beta is active (checkout 9.5 to activate it).
* Check `SEO` > `General` > `Features` to see whether the toggle is gone.
* Check the database in the `wp_options` table for the `wpseo_recalibration_beta` option and check whether it has been deleted by the upgrade routine after checking out this branch.

## UI changes

## Quality assurance


Fixes #12260
